### PR TITLE
Print empty board

### PR DESCRIPTION
**Pull request by cnak**

### CODE REVIEW

File: spec/tic_tac_toe/board_spec.clj [hunk 1]
```diff
@@ -0,0 +1,7 @@
+(ns tic-tac-toe.board-spec
+  (:require [speclj.core :refer :all]
+            [tic-tac-toe.board :refer :all]))
+
+(describe "a board"
+  (it "returns an empty board"
+    (should= '(- - - - - - - -) (empty-board) )))
```
Comment (c-j-j) [inline spec/tic_tac_toe/board_spec.clj:7]: Think about namespacing here, so it's obvious where the function is coming from. board/empty-board for example.

File: src/tic_tac_toe/board.clj [hunk 2]
```diff
@@ -0,0 +1,4 @@
+(ns tic-tac-toe.board)
+
+((defn empty-board []
```
Comment (c-j-j) [inline src/tic_tac_toe/board.clj]: Double brackets?

Comment (cnak) [inline src/tic_tac_toe/board.clj]: The round brackets (parenthesis) ?

File: spec/tic_tac_toe/board_spec.clj (hunk 1 above)
Comment (cnak) [inline spec/tic_tac_toe/board_spec.clj:7]: within the test? if I  type   

``` clojure
(should= '(- - - - - - - -) (board/empty-board) )
```

I get a stack track

Comment (c-j-j) [inline spec/tic_tac_toe/board_spec.clj:7]: you may need to give board an alias.. (my clojure is a little rusty so it'll be worth checking the various ways to import in clojure)

**Pull request merged**